widgets/src/i18n/locales/it.json

```diff
@@ -13,7 +13,9 @@
   },
   "config": {
     "knowledgeSource": "Knowledge Source",
+    "process": "Process",
     "selectConnections": "Select one of the Kie server connections.",
+    "selectBpmProcess": "Select one BPM Process.",
```

Comment:
Shouldn't these be in Italian?

Reply:
All of the IT translations are English at the moment, should be translated correctly at the end